src/com/example/ui.cljs

```diff
@@ -131,29 +131,66 @@
                  (p :.max-w-prose.mb-5.px-4
                     "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")))))
 
+(defn is-today? [date-str]
+  (let [today (js/Date.)
+        date (js/Date. date-str)]
+    (and (= (.getDate today) (.getDate date))
+         (= (.getMonth today) (.getMonth date))
+         (= (.getFullYear today) (.getFullYear date)))))
+
+(defsc Dashboard [this {:keys [readiness-quiz current-user] :root/keys [response]}]
+  {:route-segment ["dashboard"]
+   :query [{:readiness-quiz [:quiz/id]}
+           {[:root/response '_] [:response/total]}
+           {:current-user [{:responses [:response/id :response/total :response/created-at]}]}]
```

Comment:
pathom is really nice... since we have `:current-user` set up already, and the `responses` resolver takes a `user-id` as input, we can ask for the current users responses and pathom will stitch them together for us.